src/components/z-combobox/styles.css

```diff
@@ -41,11 +41,8 @@
   outline-offset: -2px;
 }
 
-:host > div.open > div.header:focus:focus-visible {
-  box-shadow: none;
-}
-
 :host > div > div.header:focus:focus-visible {
+  z-index: 16;
```

Comment:
Personalmente non starei a inventarmi un numero a caso, ma metterei il numero subito successivo a quello a cui vogliamo "passare sopra".

Reply:
Io lo lascerei così per continuità, visto che in tutto questo componente vengono lasciati spazi tra i vari z-index